src/main/java/seedu/address/model/tutorial/UniqueTutorialList.java

```diff
@@ -79,6 +79,19 @@ public void remove(Tutorial toRemove) {
         }
     }
 
+    /**
+     * Retrieves the equivalent tutorial from the list.
+     * The tutorial must exist in the list.
+     */
+    public Tutorial getTutorial(Tutorial toGet) {
+        requireNonNull(toGet);
+        int index = internalList.indexOf(toGet);
+        if (index == -1) {
```

Comment:
Is the only invalid index -1 here? Will other index be passed such that there will be outofbounds error?

Reply:
because the indexOf method returns the index of the object if the object is present and -1 if the object is not found in the list, so thats why I check if -1 is returned